client/my-sites/sidebar/sidebar.jsx

```diff
@@ -146,11 +146,21 @@ export class MySitesSidebar extends Component {
 			return null;
 		}
 
+		let sparkline;
+		if ( ! site.jetpack && ! this.isItemLinkSelected( 'stats' ) ) {
```

Comment:
Nice! You could support Jetpack with this as well by sourcing the following from the site itself:
`/wp-admin/admin.php?page=stats&noheader&proxy&chart=admin-bar-hours-scale-2x`

That's how the sparkline in the toolbar [works](https://github.com/Automattic/jetpack/blob/7c951b628286b2ba9408ae5180b412899c7957a7/modules/stats.php#L670-L683).